recipes/dd-agent.rb

```diff
@@ -158,11 +158,12 @@ def template_vars
 
 system_probe_managed = node['datadog']['system_probe']['manage_config']
 agent_version_greater_than_6_11 = agent_major_version > 5 && (agent_minor_version.nil? || agent_minor_version > 11) || agent_major_version > 6
+agent_version_greater_than_7_26 = agent_major_version > 6 && (agent_minor_version.nil? || agent_minor_version > 26) || agent_major_version > 7
```

Comment:
```suggestion
agent_version_greater_than_7_26 = agent_major_version > 5 && (agent_minor_version.nil? || agent_minor_version > 26) || agent_major_version > 7
```
Both 6.27 and 7.27 support NPM.